src/components/pages/AddFarmer/AddFarmer.jsx

```diff
@@ -0,0 +1,160 @@
+/** @format */
```

Comment:
So far, we've used `.js` file for all our React components. For the sake of consistency, could you change this to `AddFarmer.js`? This should in no way impact that functionality. 🙏

Reply:
👍

Reply:
ok .. was meant to return it back to .js. my code formatting wasnt working well thats why I used jsx.